### PR TITLE
Translate "Importing and Exporting Components" to Telugu

### LINKED ISSUE #36: Importing and Exporting Components
URL - https://te-react-dev.vercel.app/learn/importing-and-exporting-components
Requires modification to [src/content/learn/importing-and-exporting-components.md](https://github.com/reactjs/te.react.dev/tree/main/src/content/learn/importing-and-exporting-components.md) this file.

If you want to translate this page, please comment below so I can assign you.

### PR DESCRIPTION
**Pull request by Shiva-Sai-ssb**
Closes #36

### CODE REVIEW

**Comment by Shiva-Sai-ssb** [comment]
Hey @srikanth-kandi, I have translated the content into Telugu. 
Please review it when you have the time and let me know if any changes are needed.

File: src/content/learn/importing-and-exporting-components.md [hunk 1]
```diff
@@ -104,82 +104,82 @@ img { margin: 0 10px 10px 0; height: 90px; }
...
+మీరు gallery బదులు ఒకే `Profile` చూపించాలని కోరుకుంటే ఏమిటి? మీరు `Profile` కంపోనెంట్‌ను కూడా ఎక్స్‌పోర్ట్ చేయవచ్చు. కానీ `Gallery.js` లో ఇప్పటికే డిఫాల్ట్ ఎక్స్‌పోర్ట్ ఉంది, మరియు మీరు _రెండు_ డిఫాల్ట్ ఎక్స్‌పోర్ట్స్ కలిగి ఉండలేరు. మీరు ఒక కొత్త ఫైల్‌ను డిఫాల్ట్ ఎక్స్‌పోర్ట్‌తో సృష్టించవచ్చు, లేదా `Profile` కోసం నేమ్డ్ ఎక్స్‌పోర్ట్‌ను జోడించవచ్చు. **ఒక ఫైల్‌లో కేవలం ఒక డిఫాల్ట్ ఎక్స్‌పోర్ట్ ఉండవచ్చు, కానీ అనేక నేమ్డ్ ఎక్స్‌పోర్ట్స్ ఉండవచ్చు!**
 
 <Note>
 
-To reduce the potential confusion between default and named exports, some teams choose to only stick to one style (default or named), or avoid mixing them in a single file. Do what works best for you!
+డిఫాల్ట్ మరియు నేమ్డ్ ఎక్స్‌పోర్ట్స్ మధ్య గందరగోళాన్ని తగ్గించడానికి, కొంతమంది టీములు ఒకే శైలిని మాత్రమే (డిఫాల్ట్ లేదా నేమ్డ్) ఉపయోగించడం లేదా వాటిని ఒకే ఫైల్‌లో మిళితం చేయడం నుండి తప్పించుకుంటారు. మీకు ఏది సరైనదో అది చేయండి!
```
Comment (srikanth-kandi) [inline src/content/learn/importing-and-exporting-components.md:156]:
```suggestion
డిఫాల్ట్ మరియు నేమ్డ్ ఎక్స్‌పోర్ట్స్ మధ్య సంభవించే అయోమయాన్ని తగ్గించడానికి, కొంతమంది టీములు ఒకే స్టైల్ ని (డిఫాల్ట్ లేదా నేమ్డ్) మాత్రమే అనుసరించడానికి లేదా ఒకే ఫైల్లో వాటిని కలపకుండా ఉండటానికి ఎంపిక చేసుకొంటారు. మీకు ఏది సరైనదో అది చేయండి!
```

File: src/content/learn/importing-and-exporting-components.md [hunk 2]
```diff
@@ -104,82 +104,82 @@ img { margin: 0 10px 10px 0; height: 90px; }
...
 
 </Note>
 
-First, **export** `Profile` from `Gallery.js` using a named export (no `default` keyword):
+మొదట, **ఎక్స్‌పోర్ట్** చేయండి `Profile` ని `Gallery.js` నుండి, నేమ్డ్ ఎక్స్‌పోర్ట్ ద్వారా (డిఫాల్ట్ కీవర్డ్‌ను ఉపయోగించకుండా):
```
Comment (srikanth-kandi) [inline src/content/learn/importing-and-exporting-components.md:160]:
```suggestion
మొదట,`Profile` ని `Gallery.js` నుండి **ఎక్స్‌పోర్ట్** చేయండి, నేమ్డ్ ఎక్స్‌పోర్ట్ ద్వారా (డిఫాల్ట్ కీవర్డ్‌ను ఉపయోగించకుండా):
```

File: src/content/learn/importing-and-exporting-components.md [hunk 3]
```diff
@@ -104,82 +104,82 @@ img { margin: 0 10px 10px 0; height: 90px; }
...
 
 ```js
 export function Profile() {
   // ...
 }
 ```
 
-Then, **import** `Profile` from `Gallery.js` to `App.js` using a named import (with the curly braces):
+తర్వాత, **ఇంపోర్ట్** చేయండి `Profile` ని `Gallery.js` నుండి `App.js` లో, నేమ్డ్ ఇంపోర్ట్ ద్వారా (కర్లీ బ్రేసెస్‌తో):
```
Comment (srikanth-kandi) [inline src/content/learn/importing-and-exporting-components.md:168]:
```suggestion
తర్వాత, `App.js` లో `Profile` ని `Gallery.js` నుండి **ఇంపోర్ట్** చేయండి, నేమ్డ్ ఇంపోర్ట్ ద్వారా (కర్లీ బ్రేసెస్‌తో):
```

File: src/content/learn/importing-and-exporting-components.md [hunk 4]
```diff
@@ -104,82 +104,82 @@ img { margin: 0 10px 10px 0; height: 90px; }
...
 
 ```js
 import { Profile } from './Gallery.js';
 ```
 
-Finally, **render** `<Profile />` from the `App` component:
+చివరగా, **రెండర్** చేయండి `<Profile />` ని `App` కంపోనెంట్ నుండి:
```
Comment (srikanth-kandi) [inline src/content/learn/importing-and-exporting-components.md:174]:
```suggestion
చివరగా, `App` కంపోనెంట్ నుండి `<Profile />` ని **రెండర్** చేయండి:
```

File: src/content/learn/importing-and-exporting-components.md [hunk 5]
```diff
@@ -104,82 +104,82 @@ img { margin: 0 10px 10px 0; height: 90px; }
...
 
 ```js
 export default function App() {
   return <Profile />;
 }
 ```
 
-Now `Gallery.js` contains two exports: a default `Gallery` export, and a named `Profile` export. `App.js` imports both of them. Try editing `<Profile />` to `<Gallery />` and back in this example:
+ఇప్పుడు `Gallery.js` లో రెండు ఎక్స్‌పోర్ట్ లు ఉన్నాయి: ఒక డిఫాల్ట్ Gallery ఎక్స్‌పోర్ట్, మరియు ఒక నేమ్డ్ `Profile` ఎక్స్‌పోర్ట్. `App.js` రెండింటినీ ఇంపోర్ట్ చేసుకుంటుంది. ఈ ఉదాహరణలో `<Profile />` ను `<Gallery />` గా ఎడిట్ చేసి, మళ్ళీ వెనక్కి మార్చడానికి ప్రయత్నించండి:
```
Comment (srikanth-kandi) [inline src/content/learn/importing-and-exporting-components.md:182]:
```suggestion
ఇప్పుడు `Gallery.js` లో రెండు ఎక్స్‌పోర్ట్లు ఉన్నాయి: ఒక డిఫాల్ట్ `Gallery` ఎక్స్‌పోర్ట్, మరియు ఒక నేమ్డ్ `Profile` ఎక్స్‌పోర్ట్. `App.js` రెండింటినీ ఇంపోర్ట్ చేసుకుంటుంది. ఈ ఉదాహరణలో `<Profile />` ను `<Gallery />` గా ఎడిట్ చేసి, మళ్ళీ వెనక్కి మార్చడానికి ప్రయత్నించండి:
```

File: src/content/learn/importing-and-exporting-components.md [hunk 6]
```diff
@@ -222,47 +222,47 @@ img { margin: 0 10px 10px 0; height: 90px; }
...
 | -----------      | -----------                                | -----------                               |
-| Default  | `export default function Button() {}` | `import Button from './Button.js';`     |
-| Named    | `export function Button() {}`         | `import { Button } from './Button.js';` |
+| డిఫాల్ట్  | `export default function Button() {}` | `import Button from './Button.js';`     |
+| నేమ్డ్    | `export function Button() {}`         | `import { Button } from './Button.js';` |
 
 <Hint>
 
-Don't forget to import your components where they are called. Doesn't `Gallery` use `Profile`, too?
+మీరు ఎక్కడ పిలవబడతాయో అక్కడ మీ కంపోనెంట్లను ఇంపోర్ట్ చేయడం మర్చిపోకండి. `Gallery` కూడా `Profile` ను ఉపయోగించదేమో?
```
Comment (srikanth-kandi) [inline src/content/learn/importing-and-exporting-components.md:265]:
```suggestion
మీ కంపోనెంట్లను అవి పిలవబడే చోట ఇంపోర్ట్ చేయడం మర్చిపోవద్దు. `Gallery` కూడా `Profile`ను ఉపయోగిస్తుంది కదా?
```

File: src/content/learn/importing-and-exporting-components.md [hunk 7]
```diff
@@ -313,11 +313,11 @@ img { margin: 0 10px 10px 0; height: 90px; }
 
 </Sandpack>
 
-After you get it working with one kind of exports, make it work with the other kind.
+మీరు ఒక రకమైన ఎక్స్‌పోర్ట్స్‌తో పని చేయగానే, ఇతర రకమైన ఎక్స్‌పోర్ట్స్‌తో కూడా పనిచేయాలని ప్రయత్నించండి.
```
Comment (srikanth-kandi) [inline src/content/learn/importing-and-exporting-components.md:316]:
```suggestion
మీరు ఒక రకమైన ఎక్స్‌పోర్ట్‌లతో అది పని చేసేలా చేసిన తరువాత, మరొక రకంతో కూడా పని చేసేలా ప్రయత్నించండి.
```

File: src/content/learn/importing-and-exporting-components.md [hunk 8]
```diff
@@ -313,11 +313,11 @@ img { margin: 0 10px 10px 0; height: 90px; }
...
 
 <Solution>
 
-This is the solution with named exports:
+ఇది నేమ్డ్ ఎక్స్‌పోర్ట్స్‌తో సమాధానం:
```
Comment (srikanth-kandi) [inline src/content/learn/importing-and-exporting-components.md:320]:
```suggestion
ఇది నేమ్డ్ ఎక్స్‌పోర్ట్‌లతో పరిష్కారం:
```

File: src/content/learn/importing-and-exporting-components.md [hunk 9]
```diff
@@ -367,7 +367,7 @@ img { margin: 0 10px 10px 0; height: 90px; }
 
 </Sandpack>
 
-This is the solution with default exports:
+ఇది డిఫాల్ట్ ఎక్స్‌పోర్ట్స్‌తో సమాధానం:
```
Comment (srikanth-kandi) [inline src/content/learn/importing-and-exporting-components.md:370]:
```suggestion
ఇది డిఫాల్ట్ ఎక్స్‌పోర్ట్‌లతో పరిష్కారం:
```

**Review by srikanth-kandi — CHANGES_REQUESTED** [review]
Thank you @Shiva-Sai-ssb for your contribution.

Most of the Translation is good, but need some minor changes.
Please don't blindly translate as it is, just understand the meaning from the statement and convey the same meaning in Telugu with simpler and smaller sentences without losing the context.

**Comment by Shiva-Sai-ssb** [comment]
Hi @srikanth-kandi, I've made the changes as per your suggestions and would appreciate a re-review.

> Please don't blindly translate as it is, just understand the meaning from the statement and convey the same meaning in Telugu with simpler and smaller sentences without losing the context.

Definitely, I will keep that in mind and try to translate into simpler sentences.

**Review by srikanth-kandi — APPROVED** [review]

**Pull request merged**